plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/store/AnalyticsEvent.js

```diff
@@ -37,6 +37,5 @@ Ext.define('NX.coreui.store.AnalyticsEvent', {
     }
   },
 
-  sortOnLoad: true,
```

Comment:
This property is already the default so I opted to remove it. Please speak up if you have a preference in this regard. :)